docs/install/index.rst

```diff
@@ -16,20 +16,20 @@ Gammapy and it's dependencies by executing these commands in a terminal:
 
 .. code-block:: bash
 
-    curl -O https://gammapy.org/download/install/gammapy-0.18.2-environment.yml
-    conda env create -f gammapy-0.18.2-environment.yml
+    curl -O https://docs.gammapy.org/0.19/_download/environment_0.19.yml
```

Comment:
This doesn't make sense to me, I think in the dev version we should still point to the last stable version for install, in this case v0.18.2

Reply:
You're right I'll fix this.